tests/integration/test_storage_url/test.py

```diff
@@ -55,6 +57,26 @@ def test_url_cluster():
     assert result.strip() == "1\t2\t3"
 
 
+def test_url_cluster_secure():
+    query_id = f"{uuid.uuid4()}"
+
+    METADATA_SERVER_HOSTNAME = "node_imds"
+    METADATA_SERVER_PORT = 8080
+    url = f"http://adminka:secretPasswordick@{METADATA_SERVER_HOSTNAME}:{METADATA_SERVER_PORT}"
```

Comment:
Please fix test to make it work with flaky check (drop table at the end in `finally:` section).